packages/dd-trace/test/appsec/waf/waf_context_wrapper.spec.js

```diff
@@ -75,7 +119,7 @@ describe('WAFContextWrapper', () => {
 
     wafContextWrapper.run(payload)
 
-    expect(ddwafContext.run).to.have.not.been.called
```

Comment:
to have not been called is grammatically more correct but ye apparently whoever wrote those tests didn't get it right, and this is why i hate expect() xD